### PR TITLE
vioinput_enable: add new case for virtio-mouse/tablet device attached.

### PR DESCRIPTION
**Pull request by peixiu**
new case that do reboot/shutdown/system_rest/migrate
when virtio-mouse/tablet device attached.

id: 1786304, 1786302, 1786289, 1456182, 1456176, 1456177
Signed-off-by: Peixiu Hou <phou@redhat.com>

### CODE REVIEW

**Comment by peixiu** [comment]
Tested with code, all passed. 
 (05/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.vioinput_enable.device_mouse.with_shutdown.q35: PASS (125.10 s)
 (06/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.vioinput_enable.device_mouse.with_reboot.q35: PASS (121.55 s)
 (07/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.vioinput_enable.device_mouse.with_system_reset.q35: PASS (138.83 s)
 (08/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.vioinput_enable.device_mouse.with_live_migration.q35: PASS (172.18 s)
 (09/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.vioinput_enable.device_tablet.with_shutdown.q35: PASS (127.38 s)
 (10/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.vioinput_enable.device_tablet.with_reboot.q35: PASS (121.52 s)
 (11/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.vioinput_enable.device_tablet.with_system_reset.q35: PASS (141.58 s)
 (12/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.vioinput_enable.device_tablet.with_live_migration.q35: PASS (178.86 s)

**Comment by peixiu** [comment]
based on patch https://github.com/autotest/tp-qemu/pull/1902

**Comment by peixiu** [comment]
@luckyh Please help to review this patch when you free, thanks a lot~

**Comment by peixiu** [comment]
@luckyh please help to review it when you free, it based on patch https://github.com/autotest/tp-qemu/pull/1902, thanks~

File: qemu/tests/cfg/vioinput_enable.cfg [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
     inputs = input1
     input_dev_bus_type_input1 = virtio
     driver_name = "vioinput"
     extra_driver_verify = "viohidkmdf hidclass.sys hidparse.sys"
     run_bg_flag = "before_bg_test"
+    del usb_devices
+    move_rate = 80
+    move_duration = 1
+    btns = "left right middle side extra"
+    scrolls = "wheel-up wheel-down"
```
Comment (luckyh) [inline qemu/tests/cfg/vioinput_enable.cfg]: The above four lines should be moved into the mouse and the tablet variants.

Comment (peixiu) [reply qemu/tests/cfg/vioinput_enable.cfg]: Thanks, updated~

**Comment by peixiu** [comment]
Updated the code, and tested on win2012-r2 with latest code, all passed:
 (01/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.vioinput_enable.device_keyboard.with_shutdown.q35: PASS (87.45 s)
 (02/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.vioinput_enable.device_keyboard.with_reboot.q35: PASS (80.76 s)
 (03/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.vioinput_enable.device_keyboard.with_system_reset.q35: PASS (100.36 s)
 (04/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.vioinput_enable.device_keyboard.with_live_migration.q35: PASS (275.98 s)
 (05/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.vioinput_enable.device_mouse.with_shutdown.q35: PASS (74.01 s)
 (06/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.vioinput_enable.device_mouse.with_reboot.q35: PASS (66.68 s)
 (07/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.vioinput_enable.device_mouse.with_system_reset.q35: PASS (84.50 s)
 (08/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.vioinput_enable.device_mouse.with_live_migration.q35: PASS (219.94 s)
 (09/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.vioinput_enable.device_tablet.with_shutdown.q35: PASS (75.45 s)
 (10/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.vioinput_enable.device_tablet.with_reboot.q35: PASS (66.98 s)
 (11/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.vioinput_enable.device_tablet.with_system_reset.q35: PASS (84.34 s)
 (12/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.vioinput_enable.device_tablet.with_live_migration.q35: PASS (124.50 s)

**Comment by peixiu** [comment]
@luckyh Please help to review it again, thanks a lot~

**Comment by lijinlijin** [comment]
For case logic, LGTM
Acked-by: Li Jin lijin@redhat.com

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**